lib/auth/auth_with_roles.go

```diff
@@ -2955,6 +2955,25 @@ func (a *ServerWithRoles) GetSessionEvents(namespace string, sid session.ID, aft
 		return nil, trace.Wrap(err)
 	}
 
+	// emit a session recording view event for the audit log
+	user, err := a.GetCurrentUser(context.Background())
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	if a.authServer.emitter.EmitAuditEvent(a.authServer.closeCtx, &apievents.SessionRecordingView{
```

Comment:
Can you also emit this event in `StreamSessionEvents`? (Desktop access sessions use streaming, and tsh might stream soon too)